tests/infra/ccf.py

```diff
@@ -21,6 +21,10 @@
 
 logging.getLogger("paramiko").setLevel(logging.WARNING)
 
+# TODO: Quote verification may take a long time to execute on the leader when joining
+# https://github.com/microsoft/CCF/issues/703
+JOIN_TIMEOUT = 30
```

Comment:
Should we make sure this is < to the raft election timeout, at least until we have multi-threaded execution?

Reply:
As discussed, the two timers are somewhat orthogonal for our end-to-end tests. The `JOIN_TIMEOUT` just needs to greater than the time it takes for the quote verification to execute (~25 secs for worst case).